index.js

```diff
@@ -1,24 +1,26 @@
 module.exports = loader
-module.exports.testedVersions = ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.4']
+module.exports.testedVersions = ['1.8.8', '1.9.4', '1.10.2', '1.11.2', '1.12.2', '1.13.2', '1.14.4', '1.15.2', '1.16.4', 'bedrock_1.17.10']
 
 function loader (mcVersion) {
   const mcData = require('minecraft-data')(mcVersion)
   return provider({
     Biome: require('prismarine-biome')(mcVersion),
     blocks: mcData.blocks,
+    blockStates: mcData.blockStates,
+    blocksByName: mcData.blocksByName,
     blocksByStateId: mcData.blocksByStateId,
     toolMultipliers: mcData.materials,
     shapes: mcData.blockCollisionShapes,
-    isVersionNewerOrEqualTo: mcData.type === 'bedrock' ? () => false : mcData.isNewerOrEqualTo.bind(mcData),
+    version: mcData.version,
     effectsByName: mcData.effectsByName,
     enchantmentsByName: mcData.enchantmentsByName
   })
 }
 
-function provider ({ Biome, blocks, blocksByStateId, toolMultipliers, shapes, isVersionNewerOrEqualTo, effectsByName, enchantmentsByName }) {
+function provider ({ Biome, blocks, blocksByName, blocksByStateId, blockStates, toolMultipliers, shapes, version, effectsByName, enchantmentsByName }) {
   Block.fromStateId = function (stateId, biomeId) {
     // 1.13+: metadata is completely removed and only block state IDs are used
-    if (isVersionNewerOrEqualTo('1.13')) {
+    if ((version.type === 'pc' && version['>=']('1.13')) || (version.type === 'bedrock')) {
```

Comment:
that's ok, but fyi I would rather prefer we did https://github.com/PrismarineJS/mineflayer/issues/1392 rather than having these version checks everywhere